src/librustdoc/json/mod.rs

```diff
@@ -171,10 +171,17 @@ impl<'tcx> FormatRenderer<'tcx> for JsonRenderer<'tcx> {
     /// the hashmap because certain items (traits and types) need to have their mappings for trait
     /// implementations filled out before they're inserted.
     fn item(&mut self, item: clean::Item) -> Result<(), Error> {
+        // We skip children of local blanket implementations, as we'll have already seen the actual
+        // generic impl, and the generated ones don't need documenting.
```

Comment:
nit: Actually, could you move the comment back to the `if`/

Reply:
No problem, will do